docs/user/resource-aware-updates.md

```diff
@@ -0,0 +1,500 @@
+# Resource-Aware Update Control
+
+Flight Control provides intelligent resource monitoring and update control capabilities to improve the reliability of OS updates by preventing and canceling updates when the system
+is under critical resource pressure. This feature helps ensure that updates don't fail due to insufficient storage, memory, or CPU resources.
+
+## Overview
+
+The resource-aware update control system consists of several integrated components:
+
+1. **Resource Monitoring**: Continuous monitoring of CPU, memory, and disk usage with configurable thresholds
+2. **Immediate Alert Propagation**: Real-time status updates when resource alerts are triggered
+3. **Prefetch Management**: Intelligent scheduling and management of OS image downloads
+4. **Layer Size Analysis**: Pre-download analysis of OCI image layers to estimate storage requirements
+5. **Update Control**: Prevention of updates when resources are insufficient
+
+This documentation covers how to configure and use these capabilities effectively.
+
+## Resource Monitoring Configuration
```

Comment:
This is good information, we also have details on Monitors in managing-devices.md so we just want to make sure they are unified.

- https://github.com/flightctl/flightctl/blob/main/docs/user/managing-devices.md#monitoring-device-resources